.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Create a report to help us improve the library
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Before raising, please check if somebody else has already reported your issue. -->
+
+### Describe the bug
+A clear and concise description of what the bug is.
+
+### Steps to reproduce
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Environment
+* Bugsnag version:
+* Dart version:
+* Flutter version:
+* Flutter app or native app with flutter module:
+* iOS/Android version:
+* Simulator/emulator or physical device:
```

Comment:
@lemnik – can you give this list the once-over and check you're happy with what we're asking for?

Wasn't sure about Dart version - is that known given the Flutter version?

Reply:
It looks good except for that "Dart version" - that is defined by the Flutter SDK version as @tomlongridge suggested